default.env

```diff
@@ -0,0 +1,25 @@
+# Test database
```

Comment:
Where is this file used? Is it relevant if running DTBase in some other way other than using Docker containers? If not, could it be somewhere other than the root of the repo?

Reply:
This is simply a placeholder under version control that the user can copy and rename to .env for use in the dockerfile. I might actually simply rename it as something sensible and then get the user to input the correct variables.